### ISSUE TITLE
[Bug] Incorrect recipe type for "all_the_fan_made_discs:juke_recipe"

### ISSUE THREAD

**Issue by lostdoge08** [SuperNoobYT/AllTheFanMadeDiscs#5]
The file for this disc's recipe is listed as the recipe type "minecraft:smithing", which is incorrect, it is supposed to be "minecraft:smithing_transform"

**Comment by SuperNoobYT** [comment]
yes i know i forgot to update this with the 1.20 update but i would need to make a template item, i'am not sure to keep this craft
